join-orgs-beta.js

```diff
@@ -0,0 +1,25 @@
+var tap = require('tap');
+var urlOf = require('./lib/url');
+var P = require('bluebird');
+
+require('./lib/sharedNemo').then(function(nemo) {
+  nemo.state.desiredUsername = 'test-org-' + Date.now();
```

Comment:
Ugly but workable way to configure a sub-test while leaving it able to run stand-alone.